### PR TITLE
added css code in different file

### PR DESCRIPTION
**Pull request by hafedh-hichri**

### CODE REVIEW

File: templates/base.html [hunk 1]
```diff
@@ -6,143 +6,7 @@
...
-        animation: text-flicker-out-glow 6s linear infinite both;
-        margin-top:7.50%;
-        color: #fff;
-        text-transform: none;
-        text-align: center;
-        font-family: monospace;    
-      }
-
-  </style>
+  <link type="text/css" rel="stylesheet" href="{{ url_for('static', filename='temp.css')}}">
```
Comment (lf32) [inline templates/base.html:9]: Thanks @hafedh-hichri for your contribution.

I see the code is moved to `main.css`, but here the filename is `temp.css`. Could you fix it?

**Comment by hafedh-hichri** [comment]
Thanks for pointing it out , 
Let me know if everything is working properly

File: templates/base.html [hunk 2]
```diff
@@ -6,143 +6,7 @@
...
-        animation: text-flicker-out-glow 6s linear infinite both;
-        margin-top:7.50%;
-        color: #fff;
-        text-transform: none;
-        text-align: center;
-        font-family: monospace;    
-      }
-
-  </style>
+  <link type="text/css" rel="stylesheet" href="{{ url_for('static', filename='main.css')}}">
```
Comment (lf32) [inline templates/base.html:9]: Just change the filename to `css/main.css`.

File: static/css/main.css [hunk 3]
```diff
@@ -0,0 +1,158 @@
+body {
+    width: 100vw;
+    min-height: 100vh;
+    background: url("{{ url_for('static', filename='pix/dcsite-bg.jpg')}}") no-repeat fixed;
```
Comment (lf32) [inline static/css/main.css:4]: `url('../pix/dcsite-bg.jpg') no-repeat fixed;` would be enough!

**Comment by hafedh-hichri** [comment]
i changed the url for the css file as you requested 
also i realised the link i put in the html was not pointing properly , 
so i fixed it too

**Comment by lf32** [comment]
This looks great, LGTM!

**Pull request merged**